website/content/docs/install-boundary/initialize.mdx

```diff
@@ -1,25 +1,24 @@
 ---
 layout: docs
-page_title: Running Boundary in non-dev environments
+page_title: Initialize Boundary
 description: |-
-  How to install boundary in non-dev environments.
+  Creating your first login account
 ---
 
-# Install Boundary without generated resources
+# Initialize Boundary
 
-What are generated resources? When you run `boundary dev` or `boundary database init`, Boundary automatically
-generates a number of resources to make getting started easier. Default scopes, auth methods, user, account, and
-targets are just some of the resources Boundary will generate unless you tell it not to.
+Before you initialize Boundary, you should have [initialized a database](/boundary/docs/install-boundary/configure-controllers#initialize-the-database).
```

Comment:
I think we could move this sentence below lines 12 & 13, and put it under a second level heading ## Requirements for visibility.